### DISCUSSION TITLE
mac机器用 docker部署agent服务，服务已经启动，但是【设备中心】没有对应的设备

### DISCUSSION THREAD

**Discussion by liujie3612** [SonicCloudOrg/sonic-server#442] [Q&A]
1. 
![image](https://github.com/SonicCloudOrg/sonic-server/assets/9972008/781aaa6f-5ded-4afe-a9d0-2f4fc1831b8d)
服务已经启动

2. 
![image](https://github.com/SonicCloudOrg/sonic-server/assets/9972008/f24087c3-600e-4a10-acbd-cfde9e441c97)
设备已经连接

3. 
![image](https://github.com/SonicCloudOrg/sonic-server/assets/9972008/da71e7f0-b557-46df-86f1-ba332017f67a)
设备中心没有对应的设备展示，不确定和  org.cloud.sonic.agent.tools.PHCTool      : hub is not ready. ignore... 这个报错有没关系

<img width="1195" alt="image" src="https://github.com/SonicCloudOrg/sonic-server/assets/9972008/797771ae-465a-44af-a215-b846f0e6e691">

**Comment by YeungHoiChiu** [comment]
![image](https://github.com/SonicCloudOrg/sonic-server/assets/44832826/de27eb2d-198b-44cf-8274-c933a15cac9f)
